components/centos/centos-docker-base-setup/Vagrantfile

```diff
@@ -1,11 +1,15 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-#Vagrant file for libvirt/kvm and virtualbox provider
-
-# check if plugin is installed on system
-unless Vagrant.has_plugin?("vagrant-service-manager")
-  raise "vagrant-service-manager plugin is not installed, run `vagrant plugin install vagrant-service-manager` to install the plugin."
+# Vagrant file for libvirt/kvm and virtualbox provider
+REQUIRED_PLUGINS = %w(vagrant-service-manager vagrant-registration)
+message = ->(name) { "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it." }
```

Comment:
@hferentschik function was the wrong choice of word.  Can we move it to be collocated with line 12.